src/common-events/utils/headless-cms/headlessCmsUtils.tsx

```diff
@@ -184,13 +186,34 @@ export const getDefaultCollections = (
           />
         );
       } else {
+        const items = { ...collection.items };
         const cards = getGeneralCollectionCards(
           collection,
           getRoutedInternalHref,
           currentLanguageCode
-        ).map((cardProps) => (
-          <Card key={Math.random()} {...cardProps} direction="fixed-vertical" />
-        ));
+        ).map((cardProps, i) => {
+          const item = items[i] as ArticleType;
+          const categories = item?.categories as Categories;
```

Comment:
Does a page have categories? If a carousel of pages would be created, would this work? If the collection is not a EventSearchCollection, it could e.g general collection which contains pages and articles.

Reply:
not, but it will be null for pages, and still will work